packages/delivery-node/delivery.js

```diff
@@ -3,8 +3,13 @@ const request = require('./request')
 
 module.exports = (client) => ({
   sendEvent: (event, cb = () => {}) => {
+    const body = payload.event(event, client._config.redactedKeys)
+
     const _cb = err => {
       if (err) client._logger.error(`Event failed to send…\n${(err && err.stack) ? err.stack : err}`, err)
+      if (body.length > 10e5) {
+        client._logger.warn(`Discarding over-sized event (${body.length / 10e5}) after failed delivery`)
+      }
```

Comment:
the spec doesn't actually mention logging for delivery methods that don't support persistence so I'm not sure if this is wanted cc @gareththackeray and if it is perhaps the wording needs to be slightly different. Same question for the two browser delivery methods.